### ISSUE TITLE
Rooter does not work well with vim-fugitive

### ISSUE THREAD

**Issue by amosbird** [airblade/vim-rooter#39]
When I type `:Gedit`, gvim shows:

![2015-10-09_003553](https://cloud.githubusercontent.com/assets/5085485/10373081/bfc251b4-6e1d-11e5-893e-442c2c097357.png)

**Comment by airblade** [comment]
The [isVirtualFileSystem()](https://github.com/airblade/vim-rooter/blob/f8a2c2409cc7907fc2269933f0bd3f2897096ef0/plugin/rooter.vim#L42-L44) is supposed to prevent vim-rooter running on fugitive's virtual files.

Please could you insert the following line [here](https://github.com/airblade/vim-rooter/blob/f8a2c2409cc7907fc2269933f0bd3f2897096ef0/plugin/rooter.vim#L108):

```viml
echo expand('%:p')
```

Hopefully then we'll be able to see what's going on.

**Comment by amosbird** [comment]
It's `fugitive:\\C:\Users\amosb\xxx\\0\xxx`

**Comment by airblade** [comment]
Thanks.  I think the problem is the regexp is looking for forward slashes and you have back slashes.

Please can you replace [this line](https://github.com/airblade/vim-rooter/blob/f8a2c2409cc7907fc2269933f0bd3f2897096ef0/plugin/rooter.vim#L43) with this:

```viml
  return match(expand('%:p'), '^\w\+:[\/][\/].*') != -1
```

– and let me know if it solves the problem?

**Comment by amosbird** [comment]
Great, it works!

**Comment by airblade** [comment]
Fixed in a0d39b8.
